### PR TITLE
ML links update + typo

### PR DESCRIPTION
**Pull request by trebmuh**
Sourceforge mailing lists links update + one typo.

### CODE REVIEW

File: src/gui/src/AboutDialog.cpp [hunk 1]
```diff
@@ -49,6 +49,8 @@ AboutDialog::AboutDialog(QWidget* parent)
 	about += tr( "<b>Project page</b><br>");
 	about += "http://sourceforge.net/projects/hydrogen<br><br>";
 	about += tr( "<b>Mailing lists</b><br>");
+	about += "http://lists.sourceforge.net/lists/listinfo/hydrogen-users<br>";
+	about += "http://lists.sourceforge.net/lists/listinfo/hydrogen-announce<br>";
```
Comment (elpescado) [inline src/gui/src/AboutDialog.cpp]: Those mailing lists are essentially dead.

Comment (trebmuh) [inline src/gui/src/AboutDialog.cpp]: Maybe a part of the "dead" situation is coming from the fact they are not advertised ? I'm thinking especially of the use ML here.

Comment (mauser) [inline src/gui/src/AboutDialog.cpp]: The announce mailing list was listed there until last year, i have removed it. Those mailing lists are really dead. Btw, both are still listed on the homepage, so i can't see why that they are not advertised.

Comment (trebmuh) [inline src/gui/src/AboutDialog.cpp]: Still listed on the homepage **but** in the "developer zone" which is not the place the users are going to in the first place I guess.

That said, my first intention was to correct the fact that the code here above states "Mailing lists" (note the plural form). So I checked the ML available and I found the 2 others. Then I put these in the code in this commit.

It does look that keeping those ML is not the way you guys want to take, and I've got no opinion about that. I would only suggest here that it should be consistent across the software, the documentation and the website.

So, it does look to me that we have 2 possibilities:

1. these 2 ML are to be considered as "dead", then we should:
- remove these from the "developer zone -> ML"
- remove the plural form in this commit

2. these 2 ML are not that dead, then we should:
- move these from the "developer zone -> ML" to somewhere else where the common users will feel that this information is for them

If we agree on 1), I'm then removing these from the src/gui/src/AboutDialog.cpp and will enjoy to remove the plural form in the same time, making it clearer.

Agree ?

**Comment by mauser** [comment]
Hi,

i would vote for 1) ...

**Comment by trebmuh** [comment]
OK, I updated in regards to.

What is the way to follow to ask for the 2 dead lists to be removed from http://hydrogen.popez.org/hcms/node/27 ?

Also, while we are here talking about the AboutDialog.cpp file, would you think that putting a link straight to the user forum would make sense? It would make the user to know directly where the best place to go is for a user. What do you think?

**Comment by trebmuh** [comment]
@thijz , @wolkstein :  what do you think about this? Do you think we can move forward?

**Comment by thijz** [comment]
+1 for option 1)

**Comment by trebmuh** [comment]
Sweet. Thanks thijs.

@mauser : what is the correct way for 'remove these from the "developer zone -> ML"' ? Is @wolkstein still in charge with this? Maybe should I send a message to the -devel list. Should I?

**Comment by mauser** [comment]
Hi,

i've removed the links from the website.

**Comment by trebmuh** [comment]
Sweet. Thanks @mauser . Would you mind merging this PR then?

**Comment by mauser** [comment]
Done! 

About the forum link: We're currently working on a new homepage, so the link to the forum will most certainly change in the next time. I would like to keep it off the current version, but this is of course a good idea for the future. We could also create a "visit forum" or "hydrogen community" entry in the "info" menu.

**Comment by trebmuh** [comment]
thanks for the merge.
+1 for keeping this idea while waiting for the new website to pop up

**Pull request merged**